uidefinitions/createUiDefinition.json

```diff
@@ -239,7 +239,9 @@
             "Standard_F16",
             "Standard_F2",
             "Standard_F4",
-            "Standard_F8"
+            "Standard_F8",
+            "Standard_D4als_v6",
```

Comment:
Should be all Standard_D*als_v6 as listed here https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/general-purpose/dalsv6-series?tabs=sizebasic